src/microbe_stage/Microbe.cs

```diff
@@ -904,10 +904,28 @@ public void AIThink(float delta, Random random, MicrobeAICommonData data)
     /// </returns>
     public List<(Compound Compound, Color Colour, Vector3 Target)> GetDetectedCompounds(CompoundCloudSystem clouds)
     {
+        HashSet<(Compound Compound, float Range, float MinAmount, Color Colour)> colonyCompoundDetections;
+
+        if (Colony != null)
+        {
+            colonyCompoundDetections = new HashSet<(Compound Compound, float Range, float MinAmount, Color Colour)>();
+            var colonyMicrobes = Colony.ColonyMembers;
```

Comment:
This is an unnecessary intermediate variable.

Reply:
Should be gone now.

Reply:
Yep, I saw your commit earlier.